src/signalo/core/management/commands/populate_signs_poles.py

```diff
@@ -35,12 +40,17 @@ def handle(self, *args, **options):
                 poles.append(pole_instance)
 
                 # signs
+
                 for s in range(0, signs_per_pole):
                     order = s + 1
                     signs.append(
                         Sign(
                             order=order,
                             pole=pole_instance,
+                            sign_type=random.sample(sign_types, 1)[0],
+                            azimuth=Azimuth.objects.create(
```

Comment:
most likely there will be maximum one sign per azimuth.
I think we should have a stricter definition of the the dataset.
Let's say 3 random azimuth per pole, and 4 signs per azimuth.
So you should first create the azimuth and then the signs attached to it.

Reply:
Understood